### PR TITLE
BJS2-48966, They were very nice but extremely poor

### PR DESCRIPTION
**Pull request by Maksim-13**
BJS2-48966, They were very nice but extremely poor - alpha release

### CODE REVIEW

File: src/main/java/school/faang/bjs248966/Chore.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+@AllArgsConstructor
+public class Chore implements Runnable {
+
+    private String chore;
+
+    @Override
+    public void run() {
+        try {
+            System.out.println(Thread.currentThread().getName() + " выполняет задачу: " + chore);
+            Thread.sleep(1000);
```
Comment (SemyonKrivtsov) [inline src/main/java/school/faang/bjs248966/Chore.java:14]: Лучше использовать константы, а не конкретные значения

File: src/main/java/school/faang/bjs248966/WeasleyFamily.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+            if (!executor.awaitTermination(5, TimeUnit.MINUTES)) {
+                System.out.println("Задачи не завершились за 5 минут, принудительно останавливаем...");
+                executor.shutdownNow();
+            }
+        } catch (InterruptedException e) {
+            executor.shutdownNow();
+        }
+
+    }
+
```
Comment (SemyonKrivtsov) [inline src/main/java/school/faang/bjs248966/WeasleyFamily.java:37]: Лишние строки

**Review by SemyonKrivtsov — CHANGES_REQUESTED** [review]

File: src/main/java/school/faang/bjs248966/Chore.java (hunk 1 above)
Comment (Maksim-13) [reply src/main/java/school/faang/bjs248966/Chore.java]: ок, исправил

File: src/main/java/school/faang/bjs248966/WeasleyFamily.java (hunk 2 above)
Comment (Maksim-13) [reply src/main/java/school/faang/bjs248966/WeasleyFamily.java]: ок, исправил

**Pull request merged**